Core/CoreTests/Use Cases/UploadMediaCommentTests.swift

```diff
@@ -45,6 +45,12 @@ class UploadMediaCommentTests: CoreTestCase {
         XCTAssert(upload.env === environment)
     }
 
+    func testSavePlaceholderError() {
```

Comment:
I think this test should wait and verify that the placeholder `SubmissionComment` gets saved. Since it uses `performBackgroundTask`, the test might finish before the error gets set.

I worry about this because if someone comes along and adds a new required field to `SubmissionComment`, it would be easy to forget to update this placeholder creation and `save()` will fail there and this test might still pass.